perception/compare_map_segmentation/include/compare_map_segmentation/voxel_based_approximate_compare_map_filter_nodelet.hpp

```diff
@@ -16,35 +16,56 @@
 #define COMPARE_MAP_SEGMENTATION__VOXEL_BASED_APPROXIMATE_COMPARE_MAP_FILTER_NODELET_HPP_  // NOLINT
 
 #include "pointcloud_preprocessor/filter.hpp"
+#include "voxel_grid_map_loader.hpp"
 
 #include <pcl/filters/voxel_grid.h>
 #include <pcl/search/pcl_search.h>
 
+#include <memory>
+#include <string>
 #include <vector>
 
 namespace compare_map_segmentation
 {
+
+// ******************************Static Map Loader ***************************************
+class VoxelBasedApproximateStaticMapLoader : public VoxelGridStaticMapLoader
+{
+public:
+  explicit VoxelBasedApproximateStaticMapLoader(
+    rclcpp::Node * node, double leaf_size, std::string * tf_map_input_frame, std::mutex * mutex)
+  : VoxelGridStaticMapLoader(node, leaf_size, tf_map_input_frame, mutex)
+  {
+    RCLCPP_INFO(logger_, "VoxelBasedApproximateStaticMapLoader initialized.\n");
+  }
+  bool is_close_to_map(const pcl::PointXYZ & point, const double distance_threshold) override;
+};
+
+// ******************************  Dynamic Map Loader ************************************
```

Comment:
Doxygen format is better if possible.

Reply:
@yukkysaito Thanks for your review. Since those comments seem being excessive so I removed them at https://github.com/autowarefoundation/autoware.universe/pull/3365/commits/17fcb202e5ca36789f5ed0d15b478efd7d091e66